app-web/__tests__/utils/helpers.test.js

```diff
@@ -256,4 +257,28 @@ describe('Helpers', () => {
       ).toMatchSnapshot();
     });
   });
+  describe('toggle pill unit test', () => {
+    test('if All pill toggled, none of other should be toggled', () => {
+      const filterName = 'All';
+      const activeList = ['billy', 'is', 'the', 'best'];
+      const expected = ['All'];
+      const temp = pillFilter(filterName, activeList);
+      expect(temp).toEqual(expected);
+    });
+    test('if one other pill toggled, All pill should not be toggled', () => {
+      const filterName = 'billy';
+      const activeList = ['All'];
+      const expected = ['billy'];
+      const temp = pillFilter(filterName, activeList);
+      expect(temp).not.toEqual(['All']);
+      expect(temp).toEqual(expected);
+    });
+    test('multi pills can be toggle at once', () => {
+      const filterName = 'billy';
+      const activeList = ['who', 'is', 'the', 'best'];
+      const expected = ['who', 'is', 'the', 'best', 'billy'];
+      const temp = pillFilter(filterName, activeList);
+      expect(temp).toEqual(expected);
+    });
```

Comment:
looks awesome. The one thing you are missing from the spec if 'toggling off' pills